src/js/util.js

```diff
@@ -324,6 +324,11 @@ var Util;
         insertHTMLCommand: function (doc, html) {
             var selection, range, el, fragment, node, lastNode, toReplace;
 
+            // cleanup fragment comment created by Chrome
+            html = html
```

Comment:
Should this be in the paste code?  We have a set of replacements RegExp in there it could be added to, just curious why you added it here.